apps/guardian-ui/src/admin/FederationAdmin.tsx

```diff
@@ -34,15 +34,18 @@ export const FederationAdmin: React.FC = () => {
 
   useEffect(() => {
     if (!inviteCode) return;
-    api.config(inviteCode).then(setConfig).catch(console.error);
+    api.config().then(setConfig).catch(console.error);
   }, [inviteCode, api]);
```

Comment:
Since we don't need `inviteCode`, can we just call config right away without waiting to fetch it?